test/helpers/kubectl.go

```diff
@@ -2791,14 +2791,14 @@ func (kub *Kubectl) DumpCiliumCommandOutput(ctx context.Context, namespace strin
 			}
 
 			archiveName := filepath.Join(logsPath, fmt.Sprintf("bugtool-%s", pod))
-			res = kub.ExecContext(ctx, fmt.Sprintf("mkdir -p %s", archiveName))
+			res = kub.ExecContext(ctx, fmt.Sprintf("mkdir -p \"%s\"", archiveName))
```

Comment:
Drive-by nit: Can/should we use `%q` for this purpose?

Reply:
What for? As I understand it, it [does not escape parenthesis](https://stackoverflow.com/questions/56055300/what-does-safely-escaped-with-go-syntax-mean) or enquote its argument, does it?

Reply:
Ah right and I was mistaken, the `%q` does seem to enquote the string. I'll use it then, thanks.